modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -194,9 +196,9 @@ const setupShutdown = (driver: WebdriverIO.Browser, driverApi: DriverLoader.Driv
   const driverShutdown = async (immediate?: boolean) => {
     try {
       if (immediate) {
-        driver.deleteSession();
+        await driver.deleteSession();
```

Comment:
Some of this changed on my branch #145 so maybe we need to merge that and then redo this 🤔